packages/edit-site/src/components/template-details/style.scss

```diff
@@ -10,6 +10,20 @@
 	color: $gray-700;
 }
 
+.edit-site-template-details__revert {
+	border-top: $border-width solid $gray-400;
+
+	.components-menu-item__button {
+		height: auto;
+		padding: $grid-unit-20;
+		text-align: left;
+
+		&:focus:not(:disabled) {
+			box-shadow: inset 0 0 0 1.5px var(--wp-admin-theme-color), inset 0 0 0 3px #fff;
```

Comment:
Blah, I didn't notice earlier - but `#fff` could just be `$white` to match the definition in colors.

Reply:
Oops, I noticed we added the variable in one spot and not the other.

Here, we can also use `var(--wp-admin-border-width-focus)`